be/src/runtime/task_group/task_group_manager.cpp

```diff
@@ -0,0 +1,47 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "task_group_manager.h"
+
+namespace doris::taskgroup {
+
+TaskGroupManager::TaskGroupManager() {
+    _create_default_task_group();
+    _create_poc_task_group();
+}
+TaskGroupManager::~TaskGroupManager() = default;
+
+TaskGroupPtr TaskGroupManager::get_or_create_task_group(uint64_t id) {
```

Comment:
The name of this function is misleading, because you didn't create task group here.